software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -103,5 +105,5 @@
             "duration to wait whilst waiting for a machine to finish rebooting, and thus to become available again", Duration.minutes(30));
     
     AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdpPort");
-    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor("winrmPort");
+    AttributeSensorAndConfigKey<Integer, Integer> WINRM_PORT = WinRmTool.PROP_PORT;
```

Comment:
`PortAttributeSensorAndConfigKey` works with fixed port as well, see analogous [SSH_PORT](https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/core/entity/Attributes.java#L75). But fine to leave as is for this PR.